AspNetCoreApplication/Repositories/BookCategoryRepository.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AspNetCoreApplication.Exceptions;
+using AspNetCoreApplication.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AspNetCoreApplication.Repositories
+{
+    public class BookCategoryRepository : IBookCategoryRepository
+    {
+        private readonly DataContext dataContext;
+        public BookCategoryRepository(DataContext dataContext)
+        {
+            this.dataContext = dataContext;
+        }
+        public async Task AddCategoryToBook(int bookId, int categoryId)
+        {
+            var category = await dataContext.Categories.FindAsync(categoryId);
+
+            var book = await dataContext.Books.FindAsync(bookId);
+
+            var bookCategory = new BookCategory
+            {
+                BookId = bookId,
+                Book = book,
+                CategoryId = categoryId,
+                Category = category
+            };
```

Comment:
```suggestion
            var bookCategory = new BookCategory
            {
                BookId = bookId,
                CategoryId = categoryId,
            };
```

Không cần gán cho Book với Category

Reply:
done